server/api/users/users.controller.ts

```diff
@@ -28,10 +28,31 @@ export const getUserById: RequestHandler<{id: string}, UserProfile> = async (req
   res.json(user);
 };
 
-export const getUsers: RequestHandler<UserProfile[]> = async (req, res) => {
-  const users = await User.findAll({ attributes: User.allowedFields });
+export const getUsers: RequestHandler = async (req, res) => {
+  const page = req.query.page ? parseInt(req.query.page as string, 10) : 1;
+  const limit = 20;
```

Comment:
this might be useful as a constant.  i'd move this to the top of the file and capitalize it, e.g.

```js
const LIMIT = 20;
```

(capitalizing constants is a convention)

Reply:
ah yeah, seeing more in the tests - we should export this as a constant at the top level, then import it in the tests when checking for page size (i.e. if we change hte page size from 20 to 40 we wouldn't have to update all the tests too)